src/python_testing/spec_parsing_support.py

```diff
@@ -266,6 +353,55 @@ def get_problems(self) -> list[ProblemNotice]:
 
 
 def build_xml_clusters() -> tuple[list[XmlCluster], list[ProblemNotice]]:
+    # workaround for aliased clusters not appearing in the xml. Remove this once https://github.com/csa-data-model/projects/issues/373 is addressed
+    conc_clusters = {0x040C: 'Carbon Monoxide Concentration Measurement',
+                     0x040D: 'Carbon Dioxide Concentration Measurement',
```

Comment:
ok so this is temporary ... right ?

Reply:
Yup - and it's actually close to being fixed in the scraper - just one issue with the first ID and the top level cluster.